processor/lsmintervalprocessor/processor_test.go

```diff
@@ -118,3 +120,72 @@ func TestAggregation(t *testing.T) {
 		})
 	}
 }
+
+func BenchmarkAggregation(b *testing.B) {
+	testCases := []struct {
+		name        string
+		passThrough bool
+	}{
+		{name: "sum_cumulative"},
+		{name: "sum_delta"},
+		{name: "histogram_cumulative"},
+		{name: "histogram_delta"},
+		{name: "exphistogram_cumulative"},
+		{name: "exphistogram_delta"},
+		{name: "summary_enabled"},
+		{name: "summary_passthrough", passThrough: true},
+	}
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	for _, tc := range testCases {
+		config := &Config{
+			Intervals: []IntervalConfig{
+				{
+					Duration: time.Hour,
+					Statements: []string{
```

Comment:
This will utilize OTTL, let's separate this also as a variable in the benchmark function. We can do all the benchmarks with and without OTTL.